components/events/Event.tsx

```diff
@@ -25,16 +25,6 @@ const Event = (props: Props) => {
     isOver
   } = props;
 
-  const currEventRef = useRef(null);
-
-  useEffect(() => {
```

Comment:
deleted this cause i was noticing scroll jacking on dev.. i don't think it happens in prod strangely? but eh wasn't essential anyways